### PR TITLE
Added ability to create symbol/source packages

### PR DESCRIPTION
**Pull request by DavidHayden87**
As per issue #868, I have added a symbols option to the pack command which will create symbol/source packages in addition to the regular packages, where for project templates.

### CODE REVIEW

File: src/Paket.Core/NupkgWriter.fs [hunk 1]
```diff
@@ -141,7 +141,7 @@ let corePropsDoc (core : CompleteCoreInfo) =
     let xsi = XNamespace.Get "http://www.w3.org/2001/XMLSchema-instance"
     let root =
         XElement
-            (ns + "Relationships", XAttribute(XName.Get "xmlns", ns.NamespaceName),
+            (ns + "coreProperties", XAttribute(XName.Get "xmlns", ns.NamespaceName),
```
Comment (forki) [inline src/Paket.Core/NupkgWriter.fs]: is this correct? /cc @mavnn

Comment (DavidHayden87) [inline src/Paket.Core/NupkgWriter.fs]: I noticed this when comparing a package created with NuGet.exe vs. one created with Paket.exe, I'm assuming "Relationships" was a copy-paste error from the code for the .rels file.

Comment (forki) [inline src/Paket.Core/NupkgWriter.fs]: ok so it's unrelated. could you please create a second PR with only this fix?

Comment (DavidHayden87) [inline src/Paket.Core/NupkgWriter.fs]: Done #1276

**Comment by DavidHayden87** [comment]
@forki Is this pull request still being considered, or do I need to create a new one without the coreProps commit?

**Comment by forki** [comment]
no it's fine, but you can rebase this one if you like. but it's not that important

**Comment by DavidHayden87** [comment]
I've done the non-trivial merge with other changes to the Pack command on master

**Comment by forki** [comment]
goat

**Comment by forki** [comment]
could you please take a look at the docs and add a bit about this feature? thx

**Comment by DavidHayden87** [comment]
Created pull request #1279 for the documentation

**Pull request merged**